### ISSUE TITLE
Invalid path used in `npx tao` on Windows prevents workspace creation

### ISSUE THREAD

**Issue by minvedacat** [nrwl/nx#4965]
## Current Behavior

During workspace creation with `yarn create nx-workspace testx` on Windows,
`npx tao new new_workspace` failed

## Expected Behavior

It should be successful workspace creation on Windows

## Steps to Reproduce

on Windows 10 with git for windows & scoop installed nodejs 15, run with git-bash:
`yarn create nx-workspace`

### Failure Logs

```sh
CREATE testx/.prettierrc
CREATE testx/apps/.gitkeep
CREATE testx/libs/.gitkeep
EINVAL: invalid argument, mkdir 'C:\C:\Users\joe5y\repos\local\testx'
node:child_process:681
          ^
Error: Command failed: npx tao new testx --no-interactive --preset=react --appName="hello world" --style=styled-jsx --linter=eslint --no-nxCloud --collection=@nrwl/workspace/collection.json --cli=nx --nxWorkspaceRoot=\"C:/Users/joe5y/repos/local\"
    at checkExecSyncError (node:child_process:681:11)
    at Object.execSync (node:child_process:718:15)
    at createApp (C:\Users\joe5y\scoop\persist\nodejs\cache\_npx\d2207cf76adb22dc\node_modules\create-nx-workspace\bin\create-nx-workspace.js:385:21)
    at C:\Users\joe5y\scoop\persist\nodejs\cache\_npx\d2207cf76adb22dc\node_modules\create-nx-workspace\bin\create-nx-workspace.js:98:29
    at processTicksAndRejections (node:internal/process/task_queues:94:5) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 3200,
  stdout: null,
  stderr: null
}
```

### Environment

- Windows 10 build 10.0.19042
- git 2.30.1.windows.1
- node 15.8.0
- npm 7.6.1

**Comment by bekos** [comment]
hi @minvedacat . This is an issue with npm@7 on Windows, and it is already fixed via d2bac51. Can you try using `npx create nx-workspace@11.5.0-beta.0`?

**Comment by minvedacat** [comment]
> hi @minvedacat . This is an issue with npm@7 on Windows, and it is already fixed via [d2bac51](https://github.com/nrwl/nx/commit/d2bac514ac988e13e71e2c552aa4c6aef1105d92). Can you try using `npx create-nx-workspace@11.5.0-beta.0`?

Confirm with a successful project creation 👍🏼
